src/java.base/share/classes/sun/nio/ch/ThreadPool.java

```diff
@@ -72,24 +68,12 @@ int poolSize() {
         return poolSize;
     }
 
-    @SuppressWarnings("removal")
     static ThreadFactory defaultThreadFactory() {
-        if (System.getSecurityManager() == null) {
-            return (Runnable r) -> {
-                Thread t = new Thread(r);
-                t.setDaemon(true);
-                return t;
-            };
-        } else {
-            return (Runnable r) -> {
-                PrivilegedAction<Thread> action = () -> {
-                    Thread t = InnocuousThread.newThread(r);
-                    t.setDaemon(true);
-                    return t;
-               };
-               return AccessController.doPrivileged(action);
-           };
-        }
+        return (Runnable r) -> {
+            Thread t = new Thread(r);
+            t.setDaemon(true);
+            return t;
+        };
```

Comment:
This is another case where we should move to InnocuousThread, not this PR of course.

Reply:
Tracked by [JDK-8345432](https://bugs.openjdk.org/browse/JDK-8345432).